cubedash/templates/dashboard.html

```diff
@@ -0,0 +1,82 @@
+{% extends "layout/base.html" %}
+
+{% block title %}Audit Dashboard{% endblock %}
+{% block head %}
+    {{ super() }}
+    <style type="text/css">
+        .sub-header {
+            line-height: 1.4em;
+        }
+
+        a {
+            font-weight: bold;
+        }
+
+        .panel {
+            line-height: 1.5em;
+        }
+        th, td{
+            padding: 5px 10px;
+        }
+        body {
+            width: 100%;
+            max-width: 100%;
+        }
+        thead tr {
+            display: block;
+        }
+        tbody {
+            display: block;
+            height: 700px;
```

Comment:
I'm not a huge fan of inner scroll panes when they are the primary content of the page. Many platforms hide scrollbars and normal scroll shortcuts don't work. (see, for example, this github diff page as an example of glorious full-page scrolling :slightly_smiling_face:. Or the [other explorer pages](https://explorer.dea.ga.gov.au/region/ls7_level1_scene/91_80)).

Reply:
Maybe just remove the fixed height, but keep the horizontal scrollable?

Reply:
i want the column heading to be always visible

Reply:
I like static headers. My use case however does not need us to visually check the page, I will be pulling the contents into a pandas data frame and diffing to ensure the 2 datacubes are in sync. So sort of ambivalent on this particular case.